src/js_realm_object.hpp

```diff
@@ -162,6 +164,42 @@ typename T::Object RealmObjectClass<T>::create_instance(ContextType ctx, realm::
     }
 }
 
+/**
+ * @brief Implements the constructor for a Realm.Object, calling the `Realm#create` instance method to create an
+ * object in the database.
+ *
+ * @note This differes from `RealmObjectClass<T>::create_instance` as it is executed when end-users construct a `new
+ * Realm.Object()` (or another user-defined class extending `Realm.Object`), whereas `create_instance` is called when
+ * reading objects from the database.
+ *
+ * @tparam T Engine specific types.
```

Comment:
`@param` and not `@tparam`?

Reply:
This was generated by my editor: It's used to describe [a template parameter](https://www.doxygen.nl/manual/commands.html#cmdtparam).